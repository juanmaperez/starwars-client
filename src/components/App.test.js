import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import App from './App'

configure({ adapter: new Adapter() })

describe('<App/>', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<App>
      <div className="test"></div>
    </App>)
  })
  it('matches the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the children passed', () => {
    expect(wrapper.find('.test').length).toBe(1)
  })
})