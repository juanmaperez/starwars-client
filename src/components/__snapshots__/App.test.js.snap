// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> matches the snapshot 1`] = `
<div
  className="App"
>
  <Navbar
    color="light"
    expand={false}
    light={true}
    tag="nav"
  >
    <NavbarBrand
      href="/"
      tag="a"
    >
      Star Wars Client
    </NavbarBrand>
  </Navbar>
  <Container
    className="pt-5"
    tag="div"
  >
    <div
      className="test"
    />
  </Container>
</div>
`;
